packages/p2p-v2/src/components/BuySellForm/BuySellFormFooter/BuySellFormFooter.tsx

```diff
@@ -4,22 +4,22 @@ import { Button, useDevice } from '@deriv-com/ui';
 type TBuySellFormFooterProps = {
     isDisabled: boolean;
     onClickCancel: () => void;
-    onSubmit?: () => void;
+    onSubmit: () => void;
 };
-const BuySellFormFooter = ({
-    isDisabled,
-    onClickCancel,
-    onSubmit = () => {
-        //do nothing
-    },
-}: TBuySellFormFooterProps) => {
+const BuySellFormFooter = ({ isDisabled, onClickCancel, onSubmit }: TBuySellFormFooterProps) => {
     const { isMobile } = useDevice();
     return (
         <div className='flex justify-end gap-[1rem]'>
             <Button onClick={onClickCancel} size='lg' textSize={isMobile ? 'md' : 'sm'} variant='outlined'>
                 Cancel
             </Button>
-            <Button disabled={isDisabled} onClick={onSubmit} size='lg' textSize={isMobile ? 'md' : 'sm'} type='submit'>
+            <Button
+                disabled={isDisabled}
+                onClick={() => onSubmit?.()}
```

Comment:
do we need to check for `onSubmit` since it's already required?

Reply:
onSubmit isn't required above, so we need to leave it like so 🙏